packages/coil-extension/src/popup/components/MonetizationAnimation.tsx

```diff
@@ -7,44 +7,16 @@ import { notNullOrUndef } from '../../util/nullables'
 const ANIMATION_INTERVAL = 1800
 
 export const MonetizeAnimation = (props: PopupProps) => {
-  const [lastPacket, setLastPacket] = useState<Date | null>(null)
-  const [animated, setAnimated] = useState<boolean | null>(false)
-  const lastPacketRef = useRef(lastPacket)
-  lastPacketRef.current = lastPacket
+  const isPaying = localStorage.getItem('isPaying') === 'true'
+  const [animated, setAnimated] = useState<boolean>(isPaying)
 
   useEffect(() => {
-    let animateTimeout: number | null
+    const loopInterval = setInterval(() => {
+      setAnimated(localStorage.getItem('isPaying') === 'true')
```

Comment:
Can we access localStorage from the props?
We have a test page for the popup which shows it in various states(multiple instances of the popup component) and using a global wrecks that.